resources/user-guides/signal-extension-and-rules-engine-integration.md

```diff
@@ -63,15 +59,15 @@ ACPCore.collectPii(piiContextData)
 {% endtab %}
 {% endtabs %}
 
-1. In Adobe Experience Platform Launch, create a Data element for the `user_email` context data key.
+1. In Adobe Experience Platform Launch, create a data element for the `user_email` context data key.
 
-![Data Element Example for Collect PII context data key](../../../.gitbook/assets/data_element_example_collect_pii.png)
+![Data Element Example for Collect PII context data key](img/data_element_example_collect_pii.png)
 
 1. In Experience Platform Launch, create a new rule for sending a postback.
 
 Create a new rule by selecting the Event Mobile Core Collect PII and the Action Mobile Core Send Postback as in the following image:
 
-![Rule example using Collect PII event and Postback action](../../../.gitbook/assets/postback_pii_token_example.png)
+![Rule example using Collect PII event and Postback action](img/postback_pii_token_example.png)
 
 1. Use the above data element in the Postback action.
```

Comment:
Please reword to:
"Select the data element that you created in step 1 for the Postback action."